packages/waku/src/lib/builder/output-cloudflare.ts

```diff
@@ -7,21 +7,18 @@ import { DIST_PUBLIC } from './constants.js';
 // XXX this can be very limited. FIXME if anyone has better knowledge.
 export const emitCloudflareOutput = async (
   rootDir: string,
-  config: ResolvedConfig,
-  serveJs: string,
+  config: ResolvedConfig
 ) => {
   const wranglerTomlFile = path.join(rootDir, 'wrangler.toml');
   if (!existsSync(wranglerTomlFile)) {
     writeFileSync(
       wranglerTomlFile,
       `
 name = "waku-project"
-main = "${config.distDir}/${serveJs}"
-compatibility_date = "2023-12-06"
-compatibility_flags = [ "nodejs_als" ]
+compatibility_date = "2024-04-05"
+compatibility_flags = ["nodejs_compat"]
```

Comment:
I was intentionally avoiding nodejs_compat. Do we require it now?

Reply:
I'm not sure tbh, just assumed it wouldn't work unless nodejs_compat is set. Let me try without it and report back.

Any particular reason you think we should avoid it?

Reply:
I want to remove nodejs dependency strictly, and cloudflare wrangler gives me errors if some nodejs dependencies remain.